spec/lib/mavenlink/story_follow_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+describe Mavenlink::StoryFollow, type: :model do
+  describe "associations" do
+    it { is_expected.to respond_to :story }
```

Comment:
We should also spec the follower association here to and add the shared model behavior:
```
it_should_behave_like "model", "story_follows"
```